ovirt_imageio/client/_ui.py

```diff
@@ -51,11 +53,17 @@ def draw(self, value, transferred, phase=None, last=False):
         return line.ljust(self._width, " ") + end
 
 
+FORMATTER = {
+    None: TextFormat,
```

Comment:
If we use better default, we don't need to support None. args.format should never be None, since we apply default values after parsing defaults.